packages/nextjs-contentful-renderer/src/utils/contentful/getDefaultNavbar/getDefaultNavbar.ts

```diff
@@ -0,0 +1,29 @@
+import { ApolloClient, gql, NormalizedCacheObject } from '@apollo/client';
+
+export const getDefaultNavbar = async (
```

Comment:
ditto

Reply:
Following the theme implementation, what I am doing here is getting the default navbar by `domain`